alert/expiring.py

```diff
@@ -0,0 +1,86 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Search for regression in a histogram dump directory produced by the
+# node exporter.
+
+import json
+import urllib2
+from datetime import datetime, timedelta
+
+from bs4 import BeautifulSoup
+from mail import send_ses
+from version_compare import version_compare
+
+HISTOGRAMS_FILE = "Histograms.json"
+EMAIL_TIME_BEFORE = timedelta(weeks=1)
+FROM_ADDR = "telemetry-alert@mozilla.com"
+
+def get_future_release_dates():
+    # scrape for future release date tables
+    response = json.loads(urllib2.urlopen("https://wiki.mozilla.org/api.php?action=parse&format=json&page=RapidRelease/Calendar").read())
+    soup = BeautifulSoup(response["parse"]["text"]["*"])
+    table = soup.find(id="Future_branch_dates").find_parent("h2").find_next_sibling("table")
+    result = {}
+    for i, row in enumerate(table.find_all("tr")):
```

Comment:
Please write a comment with the HTML structure you are expecting to find. That will make it easier to maintain in the future.